### PR TITLE
Setup  2

### PR DESCRIPTION
**Pull request by joskalenda**
# JavaScript capstone project API-based webapp
### [ ] Features and practices applied

- Use ES6 syntax.
- Use ES6 modules.
- Use webpack.
- Use the AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Perform a code review for a team member.

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,574 @@
+:root {
+  --primary: #6167f2;
+  --primary-txt: #1f2937;
+  --secondary-txt: #1f2937;
+  --pri-bd: #bc6ce9;
+
+  /* --tertiary: #; */
+
+  /* --background: #; */
+
```
Comment (Whoistolu) [inline src/css/style.css:10]: - [ ] [OPTIONAL] Kindly remove unused code in order to have a cleaner code-base

**Review by Whoistolu — APPROVED** [review]
### Hi Team,
@joskalenda and @zieeco 

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joskalenda** [comment]
Ohhhhhh thanks for approving this PR, it was a one try approval, the way I feel like OMG 😄

File: src/css/style.css (hunk 1 above)
Comment (zieeco) [reply src/css/style.css:10]: haha... Tolu my man..Thanks

**Pull request merged**